docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -188,7 +133,7 @@ rendered, the next page of the archive is fetched. A list of the fetched pages i
 
 The `limit` prop in this case stands for the number of posts being shown, not the number of fetched pages. In the same way, the `fetchNext` shows the next post, and only fetches the next page of posts if needed.
```

Comment:
"The `limit` prop in this case stands for the number of posts being shown" - is that the number of posts **per page**?

Reply:
@DAreRodz can you clarify this? I'm using mostly the descriptions from https://community.frontity.org/t/infinite-scroll-hooks/2055/32

Regarding the `limit` property we can read in one paragraph:

> The `limit` prop, in this case, stands for the number of posts being shown, not the number of fetched pages.

but then in the table, we can read the description:

> The number of pages that the hook should load automatically before switching to manual fetching.

Can you clarify the purpose of this `limit` property a bit more?

Reply:
Sure! :slightly_smiling_face: 

The content in the table is wrong (I think it was copied from `useArchiveInfiniteScroll` and not updated, sorry :sweat_smile:).

The `limit` prop specifies the number of posts that are rendered until the user should interact (e.g. clicking a button) in order to show the next post. That value is not related to the number of posts per page.

In summary, the paragraph is OK but the table should be updated.